cylc/flow/scheduler_cli.py

```diff
@@ -54,7 +54,7 @@
 
 PLAY_DOC = r"""cylc play [OPTIONS] ARGS
 
-Start a new workflow, restart a stopped workflow, or resume a paused workflow.
+Start, restart or resume a workflow.
```

Comment:
```suggestion
Start, resume, or restart a workflow.
```
(seems a more natural ordering?)